### PR TITLE
Support multiple PB listeners

### PR DESCRIPTION
**Pull request by seancribbs**
Requires basho/riak_api#23

### CODE REVIEW

File: rel/files/nodetool [hunk 1]
```diff
@@ -228,9 +228,9 @@ handoffport_exists(Terms) ->
 
 
 pbc_port_exists(Terms) ->
-    case get_env(Terms, riak_api, pb_port) of
-        undefined ->
-            {warning, "pb_port undefined, protobuffs interface not enabled"};
+    case {get_env(Terms, riak_api, pb_port), get_env(Terms, riak_api, pb)} of
+        {undefined, PB} when PB == undefined orelse PB == [] ->
```
Comment (russelldb) [inline rel/files/nodetool]: `orelse PB == {ok, []}` works for me.

**Comment by russelldb** [comment]
Warning didn't appear for an entry of

```
{pb, []}
```

Comment above fixes that. Apart from that +1.

File: rel/files/nodetool (hunk 1 above)
Comment (seancribbs) [inline rel/files/nodetool]: Aha, thanks, fixing.

**Comment by russelldb** [comment]
Yup, +1.

**Comment by jaredmorrow** [comment]
@seancribbs this needs a patch / PR to basho/riak_ee as well.

**Comment by seancribbs** [comment]
@jaredmorrow Roger, wilco

**Pull request merged**